nessie/src/test/java/org/apache/iceberg/nessie/TestNamespace.java

```diff
@@ -132,6 +133,15 @@ public void testSettingProperties() {
         .hasMessage("Namespace does not exist: unknown");
   }
 
+  @Test
+  public void testEmptyNamespace() {
+    Assertions.assertThatThrownBy(
+            () -> catalog.createNamespace(Namespace.empty(), Collections.emptyMap()))
```

Comment:
you might also want to add a check when an empty namespace is dropped

Reply:
and maybe also set/remove properties on an empty namespace

Reply:
good suggestion. same problem exists for these also. Need to handle them gracefully.